arg_scanner/bin/arg-scanner

```diff
@@ -22,6 +24,13 @@ option_parser = OptionParser.new do |opts|
   opts.on("--no-local", "local source treatment: ignore, do not send data from local sources") do
     options.no_local = true
   end
+  opts.on("--no-type-tracker", "disable type tracker") do
+    options.no_type_tracker = true
+  end
+  opts.on("--state-tracker-dir=[Dir]", String, "enable state tracker and specify it output directory") do |dir|
```

Comment:
one should specify somewhere that absence of such property disables state tracking